.github/workflows/integration-tests.yml

```diff
@@ -63,6 +63,27 @@ jobs:
       job-tag: ${{ inputs.job-tag }}
     secrets: inherit
 
+  arm-integration-tests:
+    uses: ./.github/workflows/integration-tests-vm-type.yml
+    if: github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'run-multiarch-builds')
+    strategy:
+      # ensure that if one part of the matrix fails, the
+      # rest will continue
+      fail-fast: false
+      matrix:
+        vm_type:
+          - rhel-arm64
+          - ubuntu-os-arm64
+          - sles-arm64
+          - fcarm
+
+    with:
+      vm_type: ${{ matrix.vm_type }}
+      collector-tag: ${{ inputs.collector-tag }}
+      collector-qa-tag: ${{ inputs.collector-qa-tag }}
+      job-tag: ${{ inputs.job-tag }}
+    secrets: inherit
+
   notify:
```

Comment:
I think we'll need to add arm to the needs for this

Reply:
Yeah, I missed this one, great catch!